docs/publishing/sections/4_analytics_portfolio_site.md

```diff
@@ -9,3 +9,105 @@ For these situations, the [Jupyter Book-based](https://jupyterbook.org/en/stable
 present in the data-analyses repo is your friend.
 
 You can find the Cal-ITP Analytics Portfolio at [analysis.calitp.org](https://analysis.calitp.org).
+
+## Setup:
+Before executing the build, there are a few prior steps you need to do.
+
+1. Set up netlify key:
+    * install netlify: `npm install -g netlify-cli`
+    * Navigate to your main directory
+    * Edit your bash profile:
+        * enter `vi ~/.bash_profile` in the command line
```

Comment:
thoughts on pointing people towards `nano` instead?

Reply:
Do we have `nano` in the Hub? I thought we had to use `vi` because we didn't have anything else.